sparsity/sparse_frame.py

```diff
@@ -108,9 +108,10 @@ def todense(self, pandas=True):
         else:
             dense = np.empty(shape=(0, len(self.columns)))
 
-        if self.shape[0] == 1 or self.shape[1] == 1:
-            dense = dense.reshape(-1)
-        if pandas == True:
+        if pandas:
+            if self.shape[0] == 1 or self.shape[1] == 1:
+                dense = dense.reshape(-1)
+
```

Comment:
Yes I understand you would need to write `sf.multiply(sf['col1'].toarray().reshape(1,-1))` but then it is very clear along which axis you are multiplying. You could also do `sf.multiply(sf['col1])` to multiply along axis=1 and `sf.multiply(sf['col1].transpose())` to multiply along axis=0, but chaning `to_array` to not be one dimensional is a bad idea I think